### PR TITLE
Add/container block alignment theme changes

### PR DESCRIPTION
**Pull request by getdave**
Secret alignment sauce

### CODE REVIEW

File: packages/block-library/src/section/theme.scss [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+			width: 100%;
+		}
+	}
+
+	// Wide
+	.wp-block[data-align="wide"] {
+		width: 100%;
+
+		@include break-small() {
+			padding-right: $block-padding*4 + 1px;
```
Comment (kjellr) [inline packages/block-library/src/section/theme.scss:105]: These `1px` values are weird. Are they also due to `$borderwidth`? We should figure out why they need to be added in, and at the very least, add a code comment to explain.

File: packages/block-library/src/section/theme.scss [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+		@include break-xlarge() {
+			width: 100%;
+			// max-width: calc((6 * (100vw / 12 ) ) - #{$block-padding});
+		}
+	}
+
+	// Wide
+	.wp-block[data-align="wide"] {
+		padding-left: $block-padding*2 + 2px;
```
Comment (kjellr) [inline packages/block-library/src/section/theme.scss:61]: @getdave I _think_ the 2px values around here are due to the left + right block borders? If I'm right, you can probably swap these out for `($borderwidth*2)` to be more clear.

File: packages/block-library/src/section/theme.scss [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+
+/**
+ * Section Full Alignment
+ */
+.wp-block[data-type="core/#{$block-name}"][data-align="full"] {
+
+	.editor-inner-blocks {
+		padding-left: 46px;
```
Comment (kjellr) [inline packages/block-library/src/section/theme.scss:78]: For the final PR, it'd be excellent to spell out what these pixel measurements consist of. (I'm not totally sure myself 😄)

File: packages/block-library/src/section/theme.scss [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+	// Cancel edge padding when filling the viewport
+	.wp-block-#{$block-name} {
+		padding-left: 0;
+		padding-right: 0;
+	}
+
+	// Standard
+	.wp-block:not([data-align="wide"]):not([data-align="full"]) {
+		@include break-wide() {
+			padding-left: 42px; // why less
```
Comment (kjellr) [inline packages/block-library/src/section/theme.scss:91]: Same for these pixel measurements.

**Comment by getdave** [comment]
Now closed as code merged into main repo branchhttps://github.com/WordPress/gutenberg/pull/13964

**Pull request closed without merging**